### PR TITLE
Update grpc releases + sidebar updates + wrapper types updates

### PR DESCRIPTION
**Pull request by carsonfarmer**
Signed-off-by: Carson Farmer <carson.farmer@gmail.com>

## Description

Very minor update to include the updated method name (abort -> discard). All other API changes are not exposed.

### CODE REVIEW

**Review by andrewxhill — APPROVED** [review]

File: packages/buckets/src/api/index.ts [hunk 1]
```diff
@@ -80,23 +79,37 @@ export type RootObject = {
...
  */
-export type ListPathItemObject = {
+export type PathItemObject = {
   cid: string
   name: string
   path: string
   size: number
   isDir: boolean
-  itemsList: Array<ListPathItemObject>
+  itemsList: Array<PathItemObject>
```
Comment (andrewxhill) [inline packages/buckets/src/api/index.ts:103]: i think this should just be `items` now too

File: packages/buckets/src/api/index.ts [hunk 2]
```diff
@@ -167,10 +180,12 @@ const convertPathItem = (item: ListPathItem): ListPathItemObject => {
     size: item.getSize(),
     isDir: item.getIsDir(),
     itemsList: list ? list.map(convertPathItem) : [],
+    itemsCount: item.getItemsCount(),
```
Comment (andrewxhill) [inline packages/buckets/src/api/index.ts:183]: same, 'items'

Comment (andrewxhill) [reply packages/buckets/src/api/index.ts:183]: oop, wrong line. itemslist => item

File: packages/buckets/src/api/index.ts [hunk 3]
```diff
@@ -80,23 +79,37 @@ export type RootObject = {
...
+export type PathItemObject = {
   cid: string
   name: string
   path: string
   size: number
   isDir: boolean
-  itemsList: Array<ListPathItemObject>
+  itemsList: Array<PathItemObject>
+  itemsCount: number
+  metadata?: Metadata
```
Comment (andrewxhill) [inline packages/buckets/src/api/index.ts:105]: these metadata are looking a bit ugly to use the client. can we retype them to just make them easy. 

```
const convertMetadata = (metadata?: Metadata): PathMetadata | undefined => {
  if (!metadata) return
  const roles = metadata.getRolesMap()
  const typedRoles = new Map()
  roles.forEach((entry, key) => typedRoles.set(key, entry))
  const response: PathMetadata = {
    updatedAt: metadata.getUpdatedAt(),
    roles: typedRoles,
  }

  return response
}
```

Comment (andrewxhill) [reply packages/buckets/src/api/index.ts:105]: or maybe you have something nicer there. it's just the `PathAccessRoleMap` doesn't appear to act like a regular map created from scratch. you get some type mismatch :/ maybe not a huge deal. take a look though

Comment (andrewxhill) [reply packages/buckets/src/api/index.ts:105]: oop, i see. you gave that a nice type too. so maybe okay!!

File: packages/buckets/src/index.ts [hunk 4]
```diff
@@ -1,25 +1,24 @@
...
   ArchiveResponse,
   ArchiveStatusResponse,
   ArchiveWatchResponse,
   CreateResponse,
-  ListPathItem,
+  LinksResponse,
+  ListIpfsPathResponse,
   ListPathResponse,
   ListResponse,
+  PathItem,
```
Comment (andrewxhill) [inline packages/buckets/src/index.ts:12]: chuck in the PathMetadata

**Review by andrewxhill — APPROVED** [review]

File: packages/buckets/src/index.ts (hunk 4 above)
Comment (andrewxhill) [reply packages/buckets/src/index.ts:13]: oop, i meant MetadataObject

Comment (carsonfarmer) [reply packages/buckets/src/index.ts:13]: gotcha

Comment (carsonfarmer) [reply packages/buckets/src/index.ts:13]: metadata object is already exported from ./api

Comment (carsonfarmer) [reply packages/buckets/src/index.ts:13]: and metadata (singular) comes from buckets_pb, so I think we're good on both fronts

Comment (andrewxhill) [reply packages/buckets/src/index.ts:13]: will `import { MetadataObject } from "@textile/hub"` work?

Comment (andrewxhill) [reply packages/buckets/src/index.ts:13]: ahhhh, sorry, my bad. of course

Comment (andrewxhill) [reply packages/buckets/src/index.ts:13]: reading the wrong lines. merge away

**Pull request merged**